Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -667,6 +667,45 @@ sourceUnserviceableConfig state =
   in
     errorModalConfig'
 
+waitTimeInfoCardConfig :: ST.HomeScreenState -> RequestInfoCard.Config
+waitTimeInfoCardConfig state = let
+  isQuotes = state.data.currentSearchResultType == ST.QUOTES
+  config = RequestInfoCard.config
+  requestInfoCardConfig' = config{
+    title {
+      text = if isQuotes then getString OTP_EXPIRE_TIMER else getString WAIT_TIMER,
```

Comment:
Unify all `if isQuotes then` using a where clause. And then use it as `waitTimeConfig.title` , `waitTimeConfig.primaryText`